### DISCUSSION TITLE
"As-is" isn't a license and isn't free/open source

### DISCUSSION THREAD

**Discussion by mlysle** [opengaming/osgameclones#1544] [General]
I noticed that some projects which lack licenses are listed as licensed under "As-is". Now the words "as is" don't denote a license per se, but are words you might find written in a license.

To get down to the issue, if the project has no license, its not under some "as is" license, its proprietary software even if the source is provided. And by listing proprietary software on a page dedicated to open source games, users are misled into thinking programs allow them certain liberties which they don't have in reality.

Now, I'm not asking that unlicensed software be removed. Sometimes developers don't provide a license simply because they don't know any better, up until somebody asks them to provide one. And sites like this can help connect those developers with those somebodies.

However, I do think clarification is in order. To start with, unlicensed software should be under "N/A", not "As-is".

**Comment by cxong** [comment]
It is a good point. I don't think "N/A" is much clearer though, as it can still cause the same confusion for someone not familiar with software licenses.

Perhaps we can provide a tooltip explaining the permissions granted, kind of like the "traffic lights" on choosealicense.com.

FWIW the `As-is` wording was added in https://github.com/opengaming/osgameclones/issues/373 with no discussion, so it's worth bringing it up now.

**Comment by mlysle** [comment]
Some boilerplate I wrote:

"The author of this game has not provided it with a license. This may happen when the author does not know to include a license, or in the case of reverse-engineered code, when the author is not the actual rightsholder. You should assume all rights are reserved; access to the code is provided at the author's discretion without any right to fork, modify or republish it."

**Comment by cxong** [comment]
I also noticed the website lede is misleading, it says that all the games are open source, which is not true. I guess we can rewrite it to:

```
This site tries to gather open-source or source-available remakes of great old games in one place. If you think that something is missing from the list - please go to our GitHub repository and create an issue or even a pull request!

<second line snipped>
```

Fixing in #1722

**Comment by cxong** [comment]
https://github.com/opengaming/osgameclones/pull/1942
